### DISCUSSION TITLE
Save and close not working in pimcore 10

### DISCUSSION THREAD

**Discussion by shubhangi92** [pimcore/pimcore#10204] [General]
Hi ,

I am getting js error in pimcore 10 for save and close.

Thanks
Shubhangi

**Comment by shubhangi92** [comment]
@dvesh3 do you have idea on this , how to resolve this?

**Reply by dvesh3** [reply]
Fixed in latest version 10.1.2 https://github.com/pimcore/pimcore/pull/10207
